workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PublisherDashboard/pages/Solutions/NewSolutionFlow/pages/index.scss

```diff
@@ -0,0 +1,62 @@
+.solution-company-profile {
+	.ql-container {
```

Comment:
what means ql?

Reply:
It's the rich text editor css container.